test/Conversion/ExportChiselInterface/basic.mlir

```diff
@@ -32,7 +32,7 @@
 // CHECK-NEXT:      val ready = Output(UInt(1.W))
 // CHECK-NEXT:    }))
 // CHECK-NEXT:  }
-firrtl.circuit "Foo"  {
```

Comment:
All the non-check lines could just go in without PR.  They are just cleaning up test cases in a NFC way.